src/ott/initializers/nn/initializers.py

```diff
@@ -135,8 +135,11 @@ def update(
     """
     return self.update_impl(state, a, b)
 
-  def init_dual_a(  # noqa: D102
-      self, ot_prob: 'linear_problem.LinearProblem', lse_mode: bool
+  def init_dual_a(
+      self,
+      ot_prob: 'linear_problem.LinearProblem',
+      lse_mode: bool,
+      rng: Optional[jax.random.PRNGKey] = None
```

Comment:
`rng: jax.random.PRNGKeyArray = jax.random.PRNGKey(0)`

Reply:
This is optional right? It's not used and can be left as none

Reply:
Yes, but it's not consistent with the rest, would prefer to fix the type + do `del rng` to be explicit in the code.

Reply:
Sounds good.